### ISSUE TITLE
Add installation instruction to README

### ISSUE THREAD

**Issue by NobbZ** [agnoster/agnoster-zsh-theme#90]
I can't find an explanation about how to install this theme in a plain ZSH.

**Comment by soubhikchatterjee** [comment]
same with me. pls add instructions.

**Comment by vaibhav-jain** [comment]
@NobbZ @soubhikchatterjee any luck with the installation?

**Comment by NobbZ** [comment]
No, I didn't even try any further. I found https://github.com/bhilburn/powerlevel9k relatively quickly after posting here and used that, as it seemed to be much better documented.

Additionally I use https://github.com/zplug/zplug to manage my ZSH plugs (I might switch to another one though).

**Comment by soubhikchatterjee** [comment]
@vaibhav-jain Yes, I am using ubuntu so followed this https://gist.github.com/renshuki/3cf3de6e7f00fa7e744a
